### PR TITLE
Migrating to mapbox from g-map

### PR DESCRIPTION
**Pull request by kushthedude**
Don't hit the **UPDATE BRANCH**, Untill the PR is ready.

### CODE REVIEW

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 20
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275&bid=15810602)

File: config/environment.js [hunk 1]
```diff
@@ -97,6 +97,15 @@ module.exports = function(environment) {
     enabled: false
   };
 
+  ENV['mapbox-gl'] = {
+  	 accessToken: 'pk.eyJ1Ijoia3VzaHRoZWR1ZGUiLCJhIjoiY2s0bmczdXhlMzJidDNlbXdpY3F2enp3dSJ9.fZMstKnXl5ySzxzMajg3vQ',
+      map: {
+    style: 'mapbox://styles/mapbox/basic-v9',
```
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'style'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js [hunk 2]
```diff
@@ -28,11 +28,23 @@ export default Controller.extend({
...
+    ]
+  },
 
   actions: {
     shareEvent(event) {
       this.set('eventToShare', event);
       this.set('isShareModalOpen', true);
+    },
+        mapClicked({ target: map, point }) {
+      console.log(map, point);
```
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js [hunk 3]
```diff
@@ -97,6 +97,15 @@ module.exports = function(environment) {
...
+    zoom: 13,
+    center: [ -96.7969879, 32.7766642 ]
```
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [There should be no space before ']'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js [hunk 4]
```diff
@@ -28,11 +28,23 @@ export default Controller.extend({
...
+      }
+    ]
+  },
 
   actions: {
     shareEvent(event) {
       this.set('eventToShare', event);
       this.set('isShareModalOpen', true);
+    },
+        mapClicked({ target: map, point }) {
```
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js [hunk 5]
```diff
@@ -28,11 +28,23 @@ export default Controller.extend({
     return this.filteredEvents ? this.filteredEvents.filter(event => {return event.isFeatured}) : null;
 
   }),
+    marker: {
+    type: 'FeatureCollection',
+    features: [
```
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'features'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 3 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'center'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js [hunk 6]
```diff
@@ -28,11 +28,23 @@ export default Controller.extend({
...
+      {
+        type: 'Feature',
```
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'type'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js [hunk 7]
```diff
@@ -97,6 +97,15 @@ module.exports = function(environment) {
     enabled: false
   };
 
+  ENV['mapbox-gl'] = {
+  	 accessToken: 'pk.eyJ1Ijoia3VzaHRoZWR1ZGUiLCJhIjoiY2s0bmczdXhlMzJidDNlbXdpY3F2enp3dSJ9.fZMstKnXl5ySzxzMajg3vQ',
+      map: {
```
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js [hunk 8]
```diff
@@ -28,11 +28,23 @@ export default Controller.extend({
...
+        geometry: { type: 'Point', coordinates: [ -96.7969879, 32.7766642 ] }
```
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [There should be no space before ']'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js [hunk 9]
```diff
@@ -28,11 +28,23 @@ export default Controller.extend({
     return this.filteredEvents ? this.filteredEvents.filter(event => {return event.isFeatured}) : null;
 
   }),
+    marker: {
+    type: 'FeatureCollection',
```
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'type'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js [hunk 10]
```diff
@@ -97,6 +97,15 @@ module.exports = function(environment) {
     enabled: false
   };
 
+  ENV['mapbox-gl'] = {
+  	 accessToken: 'pk.eyJ1Ijoia3VzaHRoZWR1ZGUiLCJhIjoiY2s0bmczdXhlMzJidDNlbXdpY3F2enp3dSJ9.fZMstKnXl5ySzxzMajg3vQ',
```
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'accessToken'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 1 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js [hunk 11]
```diff
@@ -97,6 +97,15 @@ module.exports = function(environment) {
...
+      map: {
+    style: 'mapbox://styles/mapbox/basic-v9',
+    zoom: 13,
```
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 3 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js [hunk 12]
```diff
@@ -97,6 +97,15 @@ module.exports = function(environment) {
...
+    center: [ -96.7969879, 32.7766642 ]
+  }
```
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 11 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'zoom'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js [hunk 13]
```diff
@@ -28,11 +28,23 @@ export default Controller.extend({
     return this.filteredEvents ? this.filteredEvents.filter(event => {return event.isFeatured}) : null;
 
   }),
+    marker: {
```
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 8 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'geometry'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 10 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Mixed spaces and tabs.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 7 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'map'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 20
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275&bid=15810602)

File: config/environment.js (hunk 1 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 8 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [There should be no space before ']'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 7 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 12 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 13 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 8 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'geometry'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 11 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'zoom'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 3 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 10 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'accessToken'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 1 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'style'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 3 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [There should be no space before ']'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 11 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 9 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'type'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 7 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'map'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 4 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 3 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'center'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 10 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Mixed spaces and tabs.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 5 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'features'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 2 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 6 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'type'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 21
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275&bid=15810602)

File: config/environment.js (hunk 1 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 5 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'features'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 10 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'accessToken'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Mixed spaces and tabs.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 9 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'type'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 12 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 8 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'geometry'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 11 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'zoom'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/optional-features.json [hunk 14]
```diff
@@ -1,4 +1,11 @@
 {
+<<<<<<< HEAD
```
Comment (niranjan94) [inline config/optional-features.json]: Codacy Issue found: [Unexpected character ('<' (code 60)): was expecting double-quote to start field name](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 2 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Unexpected console statement.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 4 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 3 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [There should be no space before ']'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 7 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'map'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 11 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 6 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Missing space after key 'type'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 13 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 7 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: app/controllers/index.js (hunk 8 above)
Comment (niranjan94) [inline app/controllers/index.js]: Codacy Issue found: [There should be no space before ']'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 3 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'center'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

File: config/environment.js (hunk 1 above)
Comment (niranjan94) [inline config/environment.js]: Codacy Issue found: [Missing space after key 'style'.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4736275)

**Comment by kushthedude** [comment]
Messed up on local

**Pull request closed without merging**